website/docs/cloud-docs/workspaces/variables/index.mdx

```diff
@@ -152,3 +152,25 @@ When you trigger a run through the command line, Terraform Cloud applies the fol
 - **C_Variable_Set:** `KEY2`. For this run, `KEY2` equals `c`. This is a global variable set with no overwritten values.
 
 ![An example scenario demonstrating variable precedence in Terraform Cloud](/img/docs/variable-precedence-example.png)
+
+
```

Comment:
First question:
I didn't want to make the already existing precedence docs even more complicated... and was thinking to keep the two separate... since priority variable sets are for solving a specific kind of problem?

Second question:
Any varset can be set as priority, not just global -- but a global priority varset will win out over any other priority varset set with regards to precedence.


Second question:
I updated line 159 to read "all" let me know what you think!

Reply:
diagram updated -- resolving thread